Source/WebKit/Scripts/process-entitlements.sh

```diff
@@ -361,7 +361,7 @@ function ios_family_process_webcontent_shared_entitlements()
     plistbuddy Add :com.apple.private.webinspector.proxy-application bool YES
     plistbuddy Add :com.apple.private.webkit.use-xpc-endpoint bool YES
     plistbuddy Add :com.apple.runningboard.assertions.webkit bool YES
-if [[ "${PRODUCT_NAME}" != WebContentExtension ]]; then
+if [[ "${PRODUCT_BUNDLE_NAME}" != WebContentExtension ]]; then
```

Comment:
Why did we define PRODUCT_BUNDLE_NAME? AFAICT it's not a standard Xcode build setting.

I see that it's being used to set `CFBundleName` in the extension `Info.plist`s, but in the XPC service targets, the bundle name is the fully-qualified bundle identifier.

Reply:
You are right, I think we can simplify this to match the XPC service targets. There is no need to explicitly define PRODUCT_BUNDLE_NAME. Are you ok with doing this in a follow-up, since it will not actually change the CFBundleName?

Reply:
That's fine with me!